src/components/Settings/CreateLabel.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
```

Comment:
Rewrite to SASS, do not use Styled-Components as it doesn't match the stack

Reply:
We changed it from styled-components to SASS